### PR TITLE
The subject validation and keyword list

### PR DESCRIPTION
**Pull request by dakota002**

### CODE REVIEW

File: app/lib/utils.ts [hunk 1]
```diff
@@ -31,3 +31,40 @@ export function getFormDataString(formData: FormData, key: string) {
...
+  'SDSS',
+  'SGR',
+  'Swift',
+  'SWIFT',
+  'Virgo',
+  'VLA',
+  'VLBI',
+  'XRB',
+  'XTR',
+]
```
Comment (lpsinger) [inline app/lib/utils.ts:70]: Put these in circulars.server.ts, please.

File: app/routes/circulars/index.mdx [hunk 2]
```diff
@@ -41,36 +42,11 @@ Please follow the [style guidelines](/docs/styleguide). In particular, the email
...
-  <li className="grid-col-2">SDSS</li>
-  <li className="grid-col-2">SGR</li>
-  <li className="grid-col-2">Swift</li>
-  <li className="grid-col-2">SWIFT</li>
-  <li className="grid-col-2">Virgo</li>
-  <li className="grid-col-2">VLA</li>
-  <li className="grid-col-2">VLBI</li>
-  <li className="grid-col-2">XRB</li>
-  <li className="grid-col-2">XTR</li>
+  {validSubjectKeywords?.map((keyword) => (
```
Comment (lpsinger) [inline app/routes/circulars/index.mdx:45]:
```suggestion
  {validSubjectKeywords.map((keyword) => (
```

File: app/routes/circulars/submit.tsx [hunk 3]
```diff
@@ -33,13 +33,13 @@ export async function action({ request }: DataFunctionArgs) {
 }
 
 export default function Submit(props: FormProps) {
-  const defaultSubjectValid = subjectIsValid(props.subject ?? '')
+  const defaultSubjectValid = subjectLineIsValid(props.subject ?? '')
```
Comment (lpsinger) [inline app/routes/circulars/submit.tsx:36]: Why the name change?

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/circulars/submit.tsx (hunk 3 above)
Comment (dakota002) [reply app/routes/circulars/submit.tsx:36]: I wrote the new function, and when I implemented it in the submit page I saw the old function was defined there already but didnt do anything. I guess I forgot there was even a function there already. I can change the name though, that will keep the merge smaller

**Review by lpsinger — APPROVED** [review]

**Pull request merged**